README.md

```diff
@@ -1 +1,57 @@
-# Abdifatah-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+This project is a capstone project for module one in Microverse and the page is about conference.
+
+## Built With
+
+- Major languages
+  Html and css
+- Frameworks
+  no frameworks is used
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+## Getting Started
+
+You can have a copy of this repo run on your local machine by just directly cloning it
+
+> Cloning steps:
+
+1. - [ ] Open on the `Terminal`
+2. - [ ] Run `https://github.com/AbdifatahYasin1/Conference-Page.git`
+3. - [ ] cd My-portfolio-Web
+4. - [ ] Open in the IDE
+5. - [ ] open with the liver server
+
+## Author
+
+👤 **Abdifatah**
+
+- GitHub: [@AbdifatahYasin1](https://github.com/AbdifatahYasin1)
+- Twitter: [@CabdifataaxYuusuf](https://twitter.com/CabdifataaxYy)
+- LinkedIn: [Cabdifataax Yaasiin](https://www.linkedin.com/in/cabdifataax-yaasiin-69977019a/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+-Microverse were you can get quality education.
+
```

Comment:
As part of the project requirements, you are suppose to give acknowledgement or appropriate credit to  [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly update this part accordingly. :pray: